g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/translation/dao/impl/S3SingleComponentDaoImpl.java

```diff
@@ -67,12 +64,6 @@ public ComponentMessagesDTO getTranslationFromFile(ComponentMessagesDTO componen
 		if (s3util.isBundleExist(basePath, componentMessagesDTO)) {
 			componentMessagesDTO.setStatus("Translation" + TranslationQueryStatusType.FileFound.toString());
 			bunldeString = s3util.readBundle(basePath, componentMessagesDTO);
-		} else {
```

Comment:
The failure is caused by querying default translation. It's unnecessary. So remove it.